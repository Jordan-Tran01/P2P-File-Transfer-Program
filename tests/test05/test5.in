CONNECT 127.0.0.1:9856
ADDPACKAGE pkg3.bpkg
FETCH 127.0.0.1:9856 5105d1a7ffbde836fe5aaa6704ecc751857561c6073 2c8721abc90918ajh45904db002f7eb6da05f5d3031fbf0834a230d1d3691c61