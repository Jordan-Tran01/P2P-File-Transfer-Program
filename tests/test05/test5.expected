Connection established with peer
Unable to request chunk, chunk hash does not belong to package
